content/en/docs/contribute/start.md

```diff
@@ -209,16 +209,16 @@ to base your work on. Use these guidelines to make the decision:
   - Some localization teams work with a series of long-lived branches, and
     periodically merge these to `master`. This kind of branch has a name like
     dev-\<version>-\<language code>.\<team milestone>; for example:
-    `dev-{{< release-branch >}}-ja.1`.
+    `dev-1.17-ja.1`.
```

Comment:
Hi @sujaypillai .
There is another shortcode that could work in this case:
```
{{< latest-semver >}}
```

Reply:
Unless you are looking to add a different version number.

Reply:
@kbhawkey - when i use the shortcode 
```
{{< latest-semver >}}
```
the output on line 212:
`dev-1.17 -ja.1`

I get an extra space in the string which instead should be `dev-1.17-ja.1`

Reply:
👀

Reply:
OK. I created PR https://github.com/kubernetes/website/pull/19495.
Thanks for finding this issue!

Reply:
I found the issue in shortcode template - [latest-semver.html](https://github.com/kubernetes/website/blob/b92ea5765e5bd0039ab0e2aac30d64400522fd31/layouts/shortcodes/latest-semver.html#L3)

`{{- $latestSemver }}` should be replaced by `{{- $latestSemver -}}`

Reply:
Can I provide the fix here in the same pull request?